Makefile

```diff
@@ -49,15 +49,16 @@ CNI_VERSION=v0.6.0
 
 # Get version from git.
 GIT_VERSION?=$(shell git describe --tags --dirty)
+BUILD_IMAGE_ORG?=calico
 
 # By default set the CNI_SPEC_VERSION to 0.3.1 for tests.
 CNI_SPEC_VERSION?=0.3.1
 
 DIST=dist/$(ARCH)
 # Ensure that the dist directory is always created
 MAKE_SURE_DIST_EXIST := $(shell mkdir -p $(DIST))
-CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)
-CONTAINER_NAME=calico/cni
+CALICO_BUILD?=$(BUILD_IMAGE_ORG)/go-build$(ARCHTAG):$(GO_BUILD_VER)
```

Comment:
This line and the one below look incorrect for `master`.  On `master`, the arch is part of the image tag.  For example: `calico/go-build:v0.15-amd64`

In the release-v3.x branches we hadn't settled on that naming scheme yet.